src/EndpointsControllers/defaultController.js

```diff
@@ -0,0 +1,59 @@
+import { handleQuery } from '../Helpers/index.js';
+import sql from 'mysql-bricks';
+
+export class DefaultController {
+    constructor (tableName) {
+        this._tableName = tableName;
+    }
+
+    getOne () {
+        return async (req, res) => {
+            const id = req.body.id;
+            const query = sql.select()
+                .from(this._tableName)
+                .where({ id })
+                .toParams({ placeholder: '?' });
+
+            const {
+                values,
+                err
+            } = await handleQuery(query);
+
+            if (err) {
+                console.log(err);
+                return res.status(500).send(err);
+            }
+
+            if (values[0].length === 0) {
+                return res.status(404);
+            }
+
+            res.status(200).json(values[0][0]);
+        };
+    }
+
+    getMany () {
+        return async (req, res) => {
+            const fromID = req.body.fromID;
+            const amount = req.body.amount;
+
+            const query = sql.select()
+                .from(this._tableName)
+                .where(sql.gte('id', fromID))
```

Comment:
This is a black magic to me, tbh...

How can id be selected by "greater than or equal to"? I believe id should be either equal or not, nothing else.